### ISSUE TITLE
Maximum length of phrase

### ISSUE THREAD

**Issue by Davidyao99** [IDEA-Research/Grounded-Segment-Anything#378]
Great project!

May I ask if there is a maximum length of the phrase that we can input to the model? The logits seems to have a fix size of 256 and its indices seem to correspond to the phrases in the inputs.

**Comment by rentainhe** [comment]
> Great project!
> 
> I notice that the current maximum length of the input text is fixed at 256. Am I able to increase this limit without retraining the entire model? Any tips on how to do this? Thanks!

You can try to increase the input text to 512 or more by simply updating Grounding-DINO code (specially for the text-length config), but the model's performance may be influenced by the different text lengths.

**Comment by Davidyao99** [comment]
Is it enough to just change the max_text_len variable in config for this to work?
